src/components/ActivityCollector/utils.js

```diff
@@ -63,10 +63,49 @@ const isExitLink = (window, linkUrl) => {
   return true;
 };
 
+/**
+ * Reduces repeated whitespace within a string. Whitespace surrounding the string
+ * is trimmed and any occurrence of whitespace within the string is replaced with
+ * a single space.
+ * @param {string} str String to be formatted.
+ * @returns {string} Formatted string.
+ */
+const truncateWhiteSpace = str => {
```

Comment:
The current AppMeasurement_Module_ActivityMap uses this string to trim white space

`var wchar='[\\s\\n\\f\\r\\t\u0009-\u000d\u0020\u00a0\u1680\u180e\u2000-\u200a\u2028\u2029\u205f\u3000\ufeff]';
`

A number of other text editors like Word and languages can use more than just " " for white space.

Reply:
we have an assertion checking multiple spaces with the ` truncateWhiteSpace` method